src/App/Link.h

```diff
@@ -96,6 +97,10 @@ class AppExport LinkBaseExtension : public App::DocumentObjectExtension
     (LinkTransform, bool, App::PropertyBool, false, \
       "Set to false to override linked object's placement", ##__VA_ARGS__)
 
+#define LINK_PARAM_CLAIM_CHILD(...) \
```

Comment:
Of course you know my feelings about macros already, but I do recognize that in this existing code macros are how things are done, and a full re-architecting is beyond the scope of this PR.